pkgs/development/libraries/liberio/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv
+, fetchFromGitHub
+, autoreconfHook
+, systemd
+, pkg-config
+}:
+
+stdenv.mkDerivation rec {
```

Comment:
Technically the `rec` isn't necessary here, but this isn't worth doing a full review -> change -> re-review iteration for.